src/clj/util/hiccup_extensions.clj

```diff
@@ -0,0 +1,48 @@
+(ns util.hiccup-extensions
+  (:require
+    [clojure.spec.alpha :as s]
+    [clojure.string :as string]
+    [hiccup2.core :as h]))
+
+
+(s/fdef script
+        :args (s/cat :script-src (s/* string?))
+        :ret  #(instance? clojure.lang.PersistentVector %))
+
+
+(defn script
+  "Takes any number of and joins them into a javascript script tag.
```

Comment:
Any number of what?

Reply:
Updated the comment. It now reads:
`"Takes any number of arguments and joins them into a string, which is returned in a javascript script tag"`